content/pt/deployments/1.netlify.md

```diff
@@ -1,73 +1,51 @@
 
 <div class="Promo__Video">
   <a href="https://explorers.netlify.com/learn/get-started-with-nuxt/nuxt-generate-and-deploy" target="_blank">
     <p class="Promo__Video__Icon">
-      Watch a free lesson on <strong>How to deploy Nuxt to Netlify</strong> on Jamstack Explorers
+      Assista uma aula gratuita em <strong>Como fazer deploy de uma aplicação Nuxt no Netlify</strong> no Jamstack Explorers
     </p>
   </a>
 </div>
 
-## Getting Started
+## Vamos começar
 
-Press the _"New site from Git"_ button on the Netlify dashboard. Authenticate with your repository host, select a repository to deploy, and continue. You should land on step 3: _"Build options, and deploy!"_
+Pressione o botão _"New site from Git"_ ou _"Novo site a partir do Git"_ no painel de controlo do Netlify. Autentique com o hospedeiro do seu repositório, selecione um repositório para desdobrar, e continue. Você deve parar no passo 3: _"Build options, and deploy!"_ ou _"Opções de construção, e desdobrar!"_
 
 ## Configure:
 
-### For a statically generated site
+### Para um site estaticamente gerado
 
-Make sure you have `target: 'static'`in your `nuxt.config`.
+Certifique-se que você tem o `target: 'static'` dentro do seu `nuxt.config`.
 
-1. **Branch to deploy:** `main`, or which-ever branch you prefer
-1. **Build command:** `npm run generate`
-1. **Publish directory:** `dist`
+1. **Branch para fazer deploy:** `main`, ou qualquer branch que você preferir
+1. **Comando para construir:** `npm run generate`
+1. **Diretório para publicar:** `dist`
 
-### For client side rendering only
+### Para renderização apenas no lado do cliente
 
-Make sure you have `target: 'static'` and `ssr: false`in your `nuxt.config`.
+Certifique-se que você tem o `target: 'static'` e `ssr: false` dentro do seu `nuxt.config`.
 
-1. **Branch to deploy:** `main`, or which-ever branch you prefer
-1. **Build command:** `npm run generate`
-1. **Publish directory:** `dist`
+1. **Branch para fazer deploy:** `main`, ou qualquer branch que você preferir
+1. **Comando para construir:** `npm run generate`
+1. **Diretório para publicar:** `dist`
 
-For client side rendering there is a problem with refresh as by default on Netlify the site redirects to _"404 not found"_. For any pages that are not generated they will fallback to SPA mode and then if you refresh or share that link you will get Netlify's 404 page. This is because the pages that are not generated don't actually exist as they are actually a single page application so if you refresh this page you will get a 404 because the url for that page doesn't actually exist. By redirecting to the 404.html Nuxt will reload your page correctly in SPA fallback.
+Clique em _"Deploy site"_ para desdobrar imediatamente. Ao seu sítio no Netilify será atribuído uma URL aleatória e desdobrado usando o comando `nuxt generate`.
```

Comment:
There is a whole missing part here.

Reply:
@smarroufin it is okay, the missing part was added.
I am separating each content into small pull requests this time to help you guys make a more concise review.